lib/AST/ASTPrinter.cpp

```diff
@@ -3179,6 +3179,12 @@ void PrintAST::visitModuleDecl(ModuleDecl *decl) { }
 void PrintAST::visitMissingMemberDecl(MissingMemberDecl *decl) {
   Printer << "/* placeholder for ";
   recordDeclLoc(decl, [&]{ Printer << decl->getFullName(); });
+  unsigned numVTableEntries = decl->getNumberOfVTableEntries();
+  if (numVTableEntries > 0)
+    Printer << " (vtable entries: " << numVTableEntries << ")";
```

Comment:
You were going to put this behind an option, no?

Reply:
It appears that SkipMissingMemberPlaceholders is true by default, so we should only be printing these from swift-ide-test. Do you still want an option?

Reply:
Ah, didn't realize that was already there! Okay, yeah, this is fine.